onnx/inliner/inliner.cc

```diff
@@ -298,27 +466,31 @@ void InlineFunctions(ModelProto& model, FunctionResolver resolver) {
 
 // Public API implementation:
 
-void InlineLocalFunctions(ModelProto& model) {
+void InlineLocalFunctions(ModelProto& model, bool convert_version) {
   OpsetMap model_imports(model.opset_import());
   FunctionMap map;
 
+  // For every function, we check if there is a mismatch between the opset versions
+  // required for the function and the model. If there is no mismatch, we can inline
+  // this function. If there is a mismatch only for the standard ONNX domain, we
+  // can inline after version-conversion (if the version-conversion is successful).
+  // Otherwise, we cannot inline, since currently version-conversion supports only
+  // standard ONNX domain.
+
   for (auto& function : model.functions()) {
     auto mismatches = model_imports.Mismatches(function.opset_import());
+    auto iter = mismatches.find(ONNX_DOMAIN);
```

Comment:
What about other domains?

Reply:
We will not attempt version conversion if there is a mismatch for any other domain, since we don't yet have a version-converter for any other domain.